packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDayGrid..st

```diff
@@ -4,8 +4,7 @@ buildDayGrid: aBuilder
 	fields := OrderedCollection new.
 	self fillMonth.
 	1 to: 6 do: [ :row |
-		 1 to: 7 do: [ :column |
-			fields add: (self buildDayWith: aBuilder row: row column: column).
+		 1 to: 7 do: [ :column | fields add: (self buildDayWith: aBuilder row: row column: column).
```

Comment:
remove dot at end of last line

Reply:
we need the dot to be able to return fields or at least when I remove it fields turns red